### DISCUSSION TITLE
instagram cannot get "Message Request" messages

### DISCUSSION THREAD

**Discussion by ankwad** [chatwoot/chatwoot#5531] [General]
So the messenger chat feature in Meta Business suite shows new message requests, while as in chatwoot I am not able to get them

This is important, as if you operating a brand business, you would like to get messages from everyone.

I think its might be a permission issue. 

i have the follwoing permissions 
Requested: pages_manage_posts, pages_show_list, instagram_content_publish, pages_read_engagement, instagram_basic, publish_to_groups, Groups API

I have requested for these more
**Pending requests:** pages_manage_metadata, pages_messaging, instagram_manage_comments, instagram_manage_messages, Business asset user profile access

Can someone familar throw some light

Thanks in advance

**Comment by ouzbalkn** [comment]
Hello there, have you solved issue? Can you give us more details if you have done.

**Comment by sojan-official** [comment]
please follow: https://github.com/orgs/chatwoot/discussions/3397
ref: https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/instagram-channel-setup
